### ISSUE TITLE
[Accessibility: ACC-264.1] List item: Incorrect UI Type read

### ISSUE THREAD

**Issue by divyamalhotra488** [SAP/fundamental-ngx#12606]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
Type of the UI should be read out correctly to screen reader users.
Providing role, state, and value information on all user interface components enables compatibility with assistive technology, such as screen readers, screen magnifiers, and speech recognition software, used by people with disabilities.

#### Can you handle that on the application side
No

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Component Used: [Platform Table](https://sap.github.io/fundamental-ngx/#/platform/table/basic)
1. ‘Sort Ascending’ and 'Sort descending' UI are read out as ‘Link’ and
2. The filter list item is not provided with any Role.
<img width="450" alt="image" src="https://github.com/user-attachments/assets/3367d7ab-17e0-4716-a04a-3a6bd9358eec">

#### Please provide stackblitz example(s).
You can try ngx examples, issue is reproducible there.

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations.
Yes, issue is raised by an accessibility expert.

#### Did you check the documentation and the API?
Yes

#### Did you search for similar issues?
Yes

#### Is there anything else we should know?

**_IMPORTANT: Please refrain from providing links or screenshots of SAP's internal information, as this project is open-source, and its contents are accessible to anyone._**

**Comment by droshev** [comment]
We need to revisit the filtering menu item from accessibility, markup, and design point of view.

**Comment by MariaIDineva** [comment]
@divyamalhotra488 

Point 1 is not reproducible. Please test on the latest version and if you find any other issues, please open another ticket.

About point 2, can you please provide some official documentation or specification from Accessibility. We could not find such.

**Comment by droshev** [comment]
closing due not enough info
